aws-api/src/main/java/com/amplifyframework/api/aws/AppSyncGraphQLOperation.java

```diff
@@ -16,6 +16,7 @@
 package com.amplifyframework.api.aws;
 
 import android.annotation.SuppressLint;
+
```

Comment:
The current Checkstyle likes `android` and `androidx` together.

Reply:
Fixed!